cypress/integration/dishes.js

```diff
@@ -0,0 +1,36 @@
+import Common from '../obj/common';
+import DishesPage from '../obj/dishesPage';
+
+const common = new Common();
+const dishesPage = new DishesPage();
+
+describe('Lunch app. Dishes order.', function() {
+
+    it('Should not be able to place order for past days', function() {
+        cy.login();
```

Comment:
It might be better to have beforeEach function with login rather than have it in each test.